xla/service/gpu/ir_emission_utils.cc

```diff
@@ -162,6 +162,11 @@ bool IsCustomCallToTopK(const HloInstruction& hlo) {
          hlo.custom_call_target() == kTopKCustomCallTarget;
 }
 
+bool IsCustomCallToPtxKernel(const HloInstruction& hlo) {
+  return hlo.opcode() == HloOpcode::kCustomCall &&
+         hlo.custom_call_target() == "__gpu$xla.gpu.ptx";
```

Comment:
@ezhulenev I think you've been asking in the design doc for a generic call target and some mime type, right? Do you have a suggestion on how?